models/src/main/java/com/vimeo/networking2/EditSession.kt

```diff
@@ -12,6 +12,7 @@ import com.squareup.moshi.JsonClass
  * @param resultVideoHash The result video hash.
  * @param hasWatermark Whether the video has watermark.
  * @param isRated Whether the video is rated.
+ * @param minTierForMovie Whether video requires paid plan.
```

Comment:
I approved it, but I do have a question. The description sounds like this value should be a boolean instead of a string. Can you explain a little more about the potential values?

Reply:
I changed the description. Thanks